project-codeflare/multi-cluster-app-dispatcher
# [WIP] Reduce Logging noise

# Issue link


Closes #612 
# What changes have been made

Removed occurrences where an entire appwrapper is logged
Made logs at level 3 "simpler" and provided a level 4 variant that prints the original log in its entirety
Added new log which logs additional resources required for an appwrapper to be dispatched
# Verification steps



Set the logging level for MCAD to be level 3 and check that level 3 related logs are accurate.
Repeat with level 4 logs.

### To verify the required resources log
Create an AppWrapper that has a large number of resources e.g. 8 GPUs you should receive a log like this:
```
I1102 16:51:22.195583  119917 queuejob_controller_ex.go:1030] [ScheduleNext] Appwrapper 'default/raytest' requires additional resources CPU: 0.000000, Memory: 0.000000, GPU: 8.000000
```
## Checks
- Testing Strategy
